doc/rst/source/gmt.rst

```diff
@@ -11,7 +11,7 @@ gmt
 Introduction
 ------------
 
-GMT is a collection of public-domain Unix tools that allows you to
+GMT is a collection of command-line tools in the public-domain that allows you to
```

Comment:
THink we need to remove the "in the public-domain" since we have a specific copyright selected.  public domain usually means completely free of any rules, but we use GNU LGPL.  That is discussed elsewhere

Reply:
Rephrased it a bit in dc0c6a2